ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-master.yaml

```diff
@@ -59,6 +59,7 @@ tests:
         cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
         export AWS_SHARED_CREDENTIALS_FILE=${cluster_profile}/.awscred
         export KUBE_SSH_KEY_PATH=${cluster_profile}/ssh-privatekey
+        export DOCKER_RUNTIME=false
```

Comment:
Do you plan to introduce the usage of `DOCKER_RUNTIME` in  https://github.com/openshift/windows-machine-config-bootstrapper/pull/322?